test/buffer.js

```diff
@@ -12,6 +12,8 @@ describe('#buffer', () => {
 
   afterEach(done => {
     closeAll(server, statsd, false, done);
+    server = null;
+    statsd = null;
```

Comment:
A change you'll see repeating in a number of tests.  They weren't cleaning up properly.  Caused one problem with the addition of uds testing, and then I cleaned up it all.